manim/mobject/types/vectorized_mobject.py

```diff
@@ -621,7 +621,7 @@ def add_points_as_corners(self, points: np.ndarray) -> "VMobject":
             self.add_line_to(point)
         return points
 
-    def set_points_as_corners(self, points: Iterable[float]) -> "VMobject":
+    def set_points_as_corners(self, points: Sequence[float]) -> "VMobject":
```

Comment:
I think that `Sequence[float]` is not the right type to choose here.
1. Would a sequence of int not be fine as well?
2. Why is the type hint not e.g. Union[np.ndarray, List], as also here:
https://github.com/ManimCommunity/manim/blob/808c35fa3f31a1dc98ee87f11567ff9359ccb1b5/manim/mobject/mobject.py#L2006

Reply:
hey.
All I could tell was that Iterable was wrong and Sequence was "better".

There is still a lot of work to be done to fix and polish all of our type hints. I will note down for the next time to take another look at this.